packages/sdk/package.json

```diff
@@ -0,0 +1,72 @@
+{
+  "name": "@eth-optimism/sdk",
+  "version": "0.0.1",
+  "description": "[Optimism] Tools for working with Optimistic Ethereum",
+  "main": "dist/index",
+  "types": "dist/index",
+  "files": [
+    "dist/index"
+  ],
+  "scripts": {
+    "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
+    "build": "tsc -p tsconfig.build.json",
+    "clean": "rimraf dist/ ./tsconfig.build.tsbuildinfo",
+    "lint": "yarn lint:fix && yarn lint:check",
+    "lint:check": "eslint .",
+    "lint:fix": "yarn lint:check --fix",
+    "pre-commit": "lint-staged"
+  },
+  "keywords": [
+    "optimism",
+    "ethereum",
+    "sdk"
+  ],
+  "homepage": "https://github.com/ethereum-optimism/optimism-monorepo/tree/master/packages/sdk#readme",
+  "license": "MIT",
+  "author": "Optimism PBC",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ethereum-optimism/optimism-monorepo.git"
+  },
+  "devDependencies": {
```

Comment:
What is the reason behind using the `peerDependencies` for the ethers stuff?

Reply:
Whoops, this was a holdover from a previous attempt at making the package.